### ISSUE TITLE
Edit people Done button shows wrong count

### ISSUE THREAD

**Issue by waclaw66** [immich-app/immich#7341]
### The bug

The toast "Edited XX people" shows wrong count (94) of detected people (46).

![obrazek](https://github.com/immich-app/immich/assets/15554561/f82dc166-99ae-424c-946e-a60115942f28)

### The OS that Immich Server is running on

Fedora 39

### Version of Immich Server

v1.95.1

### Version of Immich Mobile App

not relevant

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
not releavant
```

### Your .env content

```Shell
not releavant
```

### Reproduction steps

```bash
1. view asset view people detected
2. click to Edit people
3. click to Done button
4. wrong count of people is shown
```

**Comment by aviv926** [comment]
@mertalev  Could it be because the rest of the face was not recognized as core?

**Comment by mertalev** [comment]
It's probably related to counting hidden people. I think this should already be fixed with #7319.

**Comment by mertalev** [comment]
Alternatively, if this issue was found on main, it could be *caused* by that issue.

**Comment by waclaw66** [comment]
It occurs on main as well, checked now. There are no hidden people.

**Comment by martabal** [comment]
It's cause by this PR #6751

**Comment by aviv926** [comment]
fixed with #7352
